lib/urldata.h

```diff
@@ -1546,6 +1547,8 @@ enum dupstring {
   STRING_SSL_CAPATH_PROXY, /* CA directory name (doesn't work on windows) */
   STRING_SSL_CAFILE_ORIG, /* certificate file to verify peer against */
   STRING_SSL_CAFILE_PROXY, /* certificate file to verify peer against */
+  STRING_SSL_CAFILE_PEM_ORIG, /* PEM certificate to verify peer against */
```

Comment:
I think you can drop the `_ORIG` suffix. I'll remove that from the other names. See #6624

Reply:
done